editor/editor_node.cpp

```diff
@@ -2367,7 +2367,7 @@ void EditorNode::_menu_option_confirm(int p_option, bool p_confirmed) {
 			if (scene) {
 				file->set_current_path(scene->get_filename());
 			};
-			file->set_title(p_option == FILE_OPEN_SCENE ? TTR("Open Scene") : TTR("Open Base Scene"));
```

Comment:
Well it's used by `case FILE_NEW_INHERITED_SCENE:`.

Reply:
Oh you're right, I didn't notice the block before not breaking, sorry!